vagrant/src/main/vagrant/servers.yaml

```diff
@@ -41,7 +41,9 @@ servers:
     box: ubuntu/wily64
     ram: 2048
     cpus: 4
-    ip: 10.10.10.100
```

Comment:
Omitting the IP is what switches to host only networking, if you specify an address here it results in Virtualbox creating a new virtual subnet to put the guest on.